gift-box-shop-frontend/pages/Shop.js

```diff
@@ -1,6 +1,30 @@
+import React,{useState,useEffect} from 'react';
 import Head from 'next/head'
 import PageWrapper from '../src/components/PageWrapper'
+import ShopMenu from '../src/components/ShopMenu';
 
 export default function Shop() {
-    return <PageWrapper>Shop</PageWrapper>
+
+    const [stepsList,setStepsList]=useState([]);  
+
+    useEffect(()=>{
+
+        //Fetch/Axios Request API
+        setStepsList(
+            [
+                {title:'1. Box Type',check:false},
```

Comment:
We should probably remove the numbers from the title and have an id inside each object with that number instead, that way we can use it as our key without needing to use the string split and we can use the id as the number as well.